src/citrine/informatics/predictors/auto_ml_predictor.py

```diff
@@ -41,7 +41,8 @@ class AutoMLPredictor(Resource['AutoMLPredictor'], Predictor, AIResourceMetadata
 
     inputs = _properties.List(_properties.Object(Descriptor), 'config.inputs')
     output = _properties.Object(Descriptor, 'output')
-    training_data = _properties.List(_properties.Object(DataSource), 'config.training_data')
+    training_data = _properties.Optional(_properties.List(_properties.Object(DataSource)),
```

Comment:
why is this optional now?

Reply:
I guess it's not optional from a serde perspective, because the backend expects something (at least an empty list)